tests/pathfinding/test_cli.py

```diff
@@ -82,7 +83,10 @@ def test_registry_address(default_cli_args):
 
     # check validation of address format
     def fails_on_registry_check(address):
-        result = runner.invoke(main, ['--registry-address', address], catch_exceptions=False)
+        result = runner.invoke(
+            main,
+            default_cli_args + ['--registry-address', address],
+            catch_exceptions=False)
```

Comment:
The closing `)` should go to the next line.

We could also think about using black for the repo in order to get rid of this manual formatting.